docs/solvedIncidentStore.md

```diff
@@ -0,0 +1,51 @@
+# Approach to Stored Analyses and Solved Incidents
+
+## Objective
+
+To ingest and permanently store all reports from Kai, using the data to determine and store solutions for fixed incidents, and to support future data extraction and fine-tuning.
```

Comment:
Should this be "store all reports from Konveyor"?

Reply:
whoops yes